content/applications/hr/lunch/orders.rst

```diff
@@ -0,0 +1,210 @@
+- :guilabel:`Received`: the product has been delivered by the vendor to the user's location.
+
+Product quantities can be adjusted by clicking the :guilabel:`➕ (plus sign)` or :guilabel:`➖ (minus
+sign)` to the left of the listed product. The product price adjusts in real-time to display the cost
+for the currently selected quantity of the product.
+
+The right side of the :guilabel:`Your Order` summary displays the various purchasing information.
+The :guilabel:`Total` amount for the entire day's lunch order is displayed. The :guilabel:`Already
+Paid` field indicates how much has been paid that day towards the :guilabel:`Total` amount. The
+:guilabel:`To Pay` field displays how much is required to pay to place the currently configured
+order.
+
+.. image:: orders/your-order.png
+   :align: center
+   :alt: The Your Orders section of the dashboard, with the purchasing information highlighted.
+
+.. tip::
+   Users can place multiple orders throughout the day, and are not restricted to placing only one
+   lunch order each day. Multiple orders can be placed due to forgetting to add items to an order,
+   or if there are multiple meals that are available to be purchased for the office, not just lunch.
+
+   Depending on the various vendors and how the vendors and products are configured, it is possible
+   to order breakfast, lunch, dinner, coffee, or snacks.
+
+Submit an order
+---------------
+
+To place the order, click the :guilabel:`Order Now` button on the right-side of the :guilabel:`Your
+Order` summary. The user is charged the amount that is displayed in the :guilabel:`To Pay` field.
+
+Once the order is placed, the tags for the items just purchased in the :guilabel:`Your Order` field
+change from orange :guilabel:`To Order` tags to red :guilabel:`Ordered` tags.
+
+Track an order
+--------------
+
+When orders have been sent to the vendors, the tags for the items in the :guilabel:`Your Order`
+summary change from red :guilabel:`Ordered` tags to blue :guilabel:`Sent` tags.
+
+Once orders have been received and verified, the tags change from blue :guilabel:`Sent` tags to
+green :guilabel:`Received` tags.
+
+Receive an order
+----------------
+
+When orders are received at the delivery location, they are confirmed, and a notification is sent
+to the employee who ordered the food.
+
+My Orders
+=========
+
+To view a full list of all orders placed in the *Lunch* app for the currently signed-in user,
+navigate to :menuselection:`Lunch app --> My Lunch: My Order History`. This navigates to the
+:guilabel:`My Orders` dashboard. The data is filtered by :guilabel:`My Orders` and grouped by
+:guilabel:`Order Date: Day`, by default.
+
+All products appear in a list view, organized by date. The list displays the :guilabel:`Order Date`,
+:guilabel:`Vendor`, :guilabel:`Product`, :guilabel:`Extras`, :guilabel:`Notes`, :guilabel:`User`,
+:guilabel:`Lunch Location`, :guilabel:`Price`, :guilabel:`Status`, and :guilabel:`Company`
```

Comment:
not showing company
![2024-04-11_10-34](https://github.com/odoo/documentation/assets/91896312/47301037-b562-41aa-afa7-095bfe4ae5a1)